c2corg_api/models/enums.py

```diff
@@ -126,6 +126,8 @@ def enum(name, types):
     'book_type', attributes.book_types)
 feed_change_type = enum(
     'feed_change_type', attributes.feed_change_types)
+default_lang = enum(
+    'default_lang', attributes.langs_priority)
```

Comment:
Using ``attributes.default_langs`` would be more consistent I think.
https://github.com/c2corg/v6_common/blob/1e3fe0991ecc8acbb0d9234f96eb71413089aa7f/c2corg_common/attributes.py#L2

In addition what about renaming the enumeration to ``langs`` instead? I don't really understand why it is "default".